android-components/config/detekt.yml

```diff
@@ -7,10 +7,17 @@ build:
     # style: 1
     # comments: 1
 
+config:
+  validation: true
+  warningsAsErrors: false
+  checkExhaustiveness: false
+  # when writing own rules with new properties, exclude the property path e.g.: 'my_rule_set,.*>.*>[my_property]'
+  excludes: ''
```

Comment:
This is just to align with the base default config https://github.com/detekt/detekt/blob/v1.22.0/detekt-core/src/main/resources/default-detekt-config.yml

My understanding of this is that it just validates the config file (`detekt.yml`).